README.md

```diff
@@ -0,0 +1,44 @@
+Project Name
+Hello Microverse project.
+
+Built With
+HTML & CSS
+
+Getting Started
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites
+A text editor eg Atom
+Setup
+Download the code in ZIP file
+Install
+Unzip the code to a folder
+Usage
+Open the folder in a text editor
+Run tests
+Test the project by running the files
+Deployment
+Deploy on Github or Cloud
+
+Authors
+👤 NduatiKagiri
+
+GitHub: @NduatiKagiri
+Twitter: @NduatiKagiri
+LinkedIn: NduatiKagiri
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
+
+Inspiration
+
+📝 License
+This project is MIT licensed.
```

Comment:
Our README needs to be as more presentable and professional as possible.
The README file is the first place where a visitor lands when visiting your repo.

- [ ] Kindly, implement back all the markdown of the README and make it have the same look as the template. Populate the sections with relevant information about the actual author and the actual repo; :+1: :+1: 

You can find the template [here](https://github.com/microverseinc/readme-template);